omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/TaskActionResource1_8.java

```diff
@@ -181,6 +185,16 @@ public DelegatingResourceDescription getCreatableProperties() throws ResourceDoe
 		return description;
 	}
 	
+	@Override
+	public Model getCREATEModel(Representation rep) {
+		ModelImpl model = (ModelImpl) super.getCREATEModel(rep);
+		model.property("tasks", new ArrayProperty());
```

Comment:
It is better to make this

    model.property("tasks", new ArrayProperty(new StringProperty()));

Please see.
https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/VisitResource1_9.java#L186